Project setup

AdrianWennberg opened the pull request.

AdrianWennberg committed:
Updated readme
Info on how to contribute to the repo and installation info

AdrianWennberg committed:
Adding initial game files
main.py runs the game, game.py is the actual game module and
config.py is the game configuration

AngryMajor reviewed (commented):
in main.py, line 9, you are reassigning then name game within the same context. is that standard way of doing it?

AdrianWennberg committed:
Fixed a naming error and made methods private

There war a naming error in the main file where the game variable was hiding the game module. I fixed it by removing the variable since it was not needed.

AngryMajor reviewed (approved):
Everything looks good

AdrianWennberg commented:
Cool, Thanks.
